src/Core/System/Snippet/BundleSnippetValidatorInterface.php

```diff
@@ -0,0 +1,16 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\System\Snippet;
+
+use Shopware\Core\Framework\Log\Package;
+
+#[Package('discovery')]
+interface BundleSnippetValidatorInterface extends SnippetValidatorInterface
+{
+    /**
+     * @param array<int, string> $bundles
+     *
+     * @return array<string, mixed>
+     */
+    public function validate(array $bundles = []): array;
```

Comment:
I am not sure, if should use the same method name. for a better deprecation path, I think a new method would be better. the other interface should be removed on the long run, right?